src/styles/blocks/_products.scss

```diff
@@ -0,0 +1,11 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+
+  margin: 0 auto 73px;
+  padding-top: 50px;
+  max-width: 1024px;
```

Comment:
13 point in checklist

Reply:
Thank you for your review! I've corrected all code according to your comments and checklist